src/msw/statbmp.cpp

```diff
@@ -333,7 +333,7 @@ void wxStaticBitmap::SetImageNoCopy( wxGDIImage* image)
             w = width;
             h = height;
 
-            ::MoveWindow(GetHwnd(), x, y, width, height, FALSE);
+            MSWMoveWindowToAnyPosition(GetHwnd(), x, y, width, height);
```

Comment:
Doesn't this change the code by repainting the window now when it wasn't (presumably for some good reason?) done before?

I think it'd be better if we just kept `bRepaint` argument in `MSWMoveWindowToAnyPosition()` for consistency with `::MoveWindow()` in any case.